[TH2-2094] Update README.MD (#7)
th2-net/th2-act-ui-backend: README.md

# Act-ui backend

# Overview

> This is a backend component for the [act-ui](https://github.com/th2-net/th2-act-ui) web app. Act-ui backend provides autocompletion data and enables communication with th2 infrastructure. To fucntion properly, conn (or codec) components need to be connected to it.

Reviewer: ```suggestion
This is a backend component for the [act-ui](https://github.com/th2-net/th2-act-ui) web app. Act-ui backend provides autocompletion data and enables communication with th2 infrastructure. To function properly, conn (or codec) components needs to be connected to it.
```

  name: act-ui-backend
spec:
  image-name: ghcr.io/th2-net/th2-act-ui-backend
  image-version: 0.3.0 # change this line if you want to use a newer version
  type: th2-rpt-data-provider
  custom-config:
    hostname: "localhost"
    port: 8080
    responseTimeout: 6000 # maximum request processing time in milliseconds

    clientCacheTimeout: 60 # cached event lifetime in milliseconds
    ioDispatcherThreadPoolSize: 10 # thread pool size for blocking database calls
        
    schemaDefinitionLink: "" # link of schema definition
    protoCompileDirectory: "src/main/resources/protobuf" # directory for compiling proto files
    namespace: "th2-namespace" # namespace for sending grpc messages

>     actTypes: ["th2-act"] # the types of services the ACTs method will look for

Reviewer: ```suggestion
    actTypes: ["th2-act"] # the types of services that ACTs method will look for
```

        cpu: 50m

```

> ### Act mode additianal configuration

Reviewer: ```suggestion
### Act mode additional configuration
```

> Act should have attached gRPC desciptors to be accessible from act-ui. To generate gescriptors add [plugin](https://github.com/th2-net/th2-box-descriptor-generator) to build script and configure CI to attach them to the docker image. You can see an example of the configured act [here](https://github.com/th2-net/th2-act-template-j).\

Reviewer: ```suggestion
Act should have attached gRPC descriptors to be accessible from act-ui. To generate descriptors add the following [plugin](https://github.com/th2-net/th2-box-descriptor-generator) to build the script and configure CI to attach them to the docker image. You can see an example of the configured act [here](https://github.com/th2-net/th2-act-template-j).\
```

> Also ensure that deployed act box type (spec -> type in yml file) is presented in act-ui-backend configuration (spec -> actTypes in yml file)

Reviewer: ```suggestion
Also ensure that the deployed act box type (spec -> type in yml file) is presented in act-ui-backend configuration (spec -> actTypes in yml file)
```
